tools/extract_text.py

```diff
@@ -1882,6 +1884,148 @@ def format_color(self, c : int) -> str:
             0x47 : "BLACK",
         }[c]
 
+# Chinese Characters ordered according to their appearance in nes_font_static.
+# This unfortunately does not appear to match any standard encoding.
+CHN_CHARS =      """你借到了一\
```

Comment:
could we share this charmap with the encoding part?

Reply:
Not really, since the encoding charmap can be changed in modding, which would break extracting vanilla.